README.md

```diff
@@ -1,2 +1,95 @@
 [![Build Status](https://travis-ci.org/pavloo/config_py.svg?branch=master)](https://travis-ci.org/pavloo/config_py)
-### TBD
+
+# Disclaimer: the library is under active development and not intended to be used right now
+
+## config_py
+*A Python package for managing application configurations based on [Convention Over Configuration](https://en.wikipedia.org/wiki/Convention_over_configuration) principle*.
+
```

Comment:
I think we need some short explanation why you need this at all - e.g. 
Sample use cases: 
- You want use different URLs based on running in production and during development 
- You want to disable or enable some features in production but keep them on in development